packages/connect-query/package.json

```diff
@@ -52,7 +52,7 @@
   "peerDependencies": {
     "@bufbuild/protobuf": "^1.3.3",
     "@connectrpc/connect": "^1.1.2",
-    "@tanstack/react-query": "4.x",
+    "@tanstack/react-query": "5.0.0 < 6.0.0",
```

Comment:
This constraint won't match version `5.0.5`, the current latest.

Reply:
Oh man, yeah I totally missed that. I'm actually weirded out that it only matches 5.0.0 🤔 , oh well, I'll update it to `5 < 6` which seems to match fine (according to https://semver.npmjs.com/)

Reply:
Strike that, 5 < 6 doesn't work the way I thought either... Maybe just `5.x` is just fine.

Reply:
We can simply use `^5.0.0` here. I recommend this tool: https://github.com/jubianchi/semver-check